### PR TITLE
Add the contribute.json endpoint.

### LINKED ISSUE #19: Add a contribute.json file
See https://www.contributejson.org/

### PR DESCRIPTION
**Pull request by Natim**
Fixes #19

### CODE REVIEW

File: pollbot/app.py [hunk 1]
```diff
@@ -6,6 +6,7 @@ def get_app(loop=None):
     app = web.Application(loop=loop)
     app.router.add_get('/', home.redirect)
     app.router.add_get('/v1/', home.index)
+    app.router.add_get('/v1/contribute.json', utilities.contribute_json)
```
Comment (magopian) [inline pollbot/app.py]: should it instead be at the root: `/contribute.json` ?

File: pollbot/views/utilities.py [hunk 2]
```diff
@@ -8,5 +8,11 @@
...
     with open(os.path.join(HERE, "..", "api.yaml")) as stream:
-        oas_spec = yaml.load(stream)
+        oas_spec = yaml.safe_load(stream)
     return web.json_response(oas_spec)
+
+
+async def contribute_json(request):
+    with open(os.path.join(HERE, "..", "contribute.yaml")) as stream:
+        contribute = yaml.safe_load(stream)
+    return web.json_response(contribute)
```
Comment (magopian) [inline pollbot/views/utilities.py]: Do you want to use a helper there too?

File: tests/test_views.py [hunk 3]
```diff
@@ -20,11 +20,18 @@ def cli(loop, test_client):
 
 async def test_oas_spec(cli):
     with open(os.path.join(HERE, "..", "pollbot", "api.yaml"), 'r') as stream:
-        oas_spec = yaml.load(stream)
+        oas_spec = yaml.safe_load(stream)
```
Comment (magopian) [inline tests/test_views.py]: Are you testing the same piece of code twice (with the `test_oas.py` file)?

File: tests/test_views.py [hunk 4]
```diff
@@ -20,11 +20,18 @@ def cli(loop, test_client):
...
     assert await resp.json() == oas_spec
 
 
+async def test_contribute_json(cli):
+    with open(os.path.join(HERE, "..", "pollbot", "contribute.yaml"), 'r') as stream:
+        contribute = yaml.safe_load(stream)
+    resp = await cli.get("/v1/contribute.json")
+    assert await resp.json() == contribute
+
+
```
Comment (magopian) [inline tests/test_views.py]: same here, use a helper to factor those out?

File: tests/test_views.py (hunk 3 above)
Comment (Natim) [inline tests/test_views.py]: Not really, one tests the content of the spec, the other one that the spec is served via HTTP.

**Pull request merged**